tests/yast2_cmd/yast_keyboard.pm

```diff
@@ -48,7 +48,7 @@ sub run {
 
     # Restore keyboard settings to english-us and verify(enter using german characters).
     if (is_sle('15-sp2+')) {
-        type_string("zast kezboard set lazout)english/us\n", wait_still_screen => 3, timeout => 7);
+        type_string("zast kezboard set lazout)english/us\n", wait_still_screen => 5, timeout => 10);
```

Comment:
Is it stable with those timeouts? As per documentation default timeout is 30 seconds with stilltime eq 0. So maybe just setting `wait_still_screen` to `5` will be enough?

Reply:
seems to be save. 
I am not sure why martin had changed the timeout but with default we made the test unnecessarily slower.

> Is it stable with those timeouts? As per documentation default timeout is 30 seconds with stilltime eq 0. So maybe just setting `wait_still_screen` to `5` will be enough?